src/app/products/components/product-table/product-table.component.spec.ts

```diff
@@ -181,17 +189,28 @@ describe('AppModule => ProductTable', () => {
       expect(document.execCommand).toHaveBeenCalledWith('copy');
     });
 
-    it('should show description tooltip over truncated text', async () => {
+    it('should call snackBar open method with expected params on Copy Url btn click', async () => {
```

Comment:
Wasn't able to write tests for snack bar using MatSnackBarHarness because when you pass `duration` config, jasmine wait until it closes and only then proceed, hence I simply can't locate it

For more info take a look at this issue: []((https://github.com/angular/components/issues/19290))